src/app/routes/public-forms.server.routes.js

```diff
@@ -207,9 +208,7 @@ module.exports = function (app) {
       }),
     }),
     forms.formById,
-    publicForms.isFormPublicCheck,
-    CaptchaMiddleware.checkCaptchaResponse,
-    publicForms.checkFormSubmissionLimitAndDeactivate,
```

Comment:
should these middleware functions also be deleted from the codebase, since they're no longer in use?

Reply:
So as it turns out the `isFormPublicCheck` and the `checkFormSubmissionLimitAndDeactivate` filters are still used in https://github.com/opengovsg/FormSG/blob/c5c88bacd97ad02c855139c8b07c55054304ddbd/src/app/routes/public-forms.server.routes.js#L132-L141 so they can't be removed. Removed `checkCaptchaResponse`

Reply:
ok, thanks for tracing the code